src/bp/ui-admin/src/telemetry/offlineServerFallback.ts

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
+import _ from 'lodash'
+import ms from 'ms'
+
+import { Schema } from '../../../common/telemetry'
+import api from '../api'
+
+import { axiosConfig } from './client'
+
+const serverUrlPayloads = '/admin/telemetry-payloads'
+const serverUrlFeedback = '/admin/telemetry-feedback'
```

Comment:
Are you using those URL in multiple places ? If yes, this would be more clean: 
```js
const endpoint = {
  payloads: '',
  feedback: ''
}
```
If it's the only place it's being used, you can just put them directly in the axios calls